src/main/java/cms/gov/madie/measure/utils/JsonUtil.java

```diff
@@ -224,12 +232,78 @@ public static List<TestCaseGroupPopulation> getTestCaseGroupPopulationsFromMeasu
               groupPopulations.add(groupPopulation);
             }
           }
+          // get Stratifications
+          JsonNode stratifications = group.get("stratifier");
+          if (stratifications != null) {
+            List<TestCaseStratificationValue> stratificationValues = new ArrayList<>();
+            AtomicInteger stratCount = new AtomicInteger(0);
+            stratifications.forEach(
+                (stratification) ->
+                    buildStratificationPopValues(
+                        group,
+                        stratificationValues,
+                        stratification,
+                        stratCount.getAndIncrement(),
+                        measurePopulationBasis));
+            groupPopulation.setStratificationValues(stratificationValues);
+          }
         }
       }
     }
     return groupPopulations;
   }
 
+  private static void buildStratificationPopValues(
+      JsonNode group,
+      List<TestCaseStratificationValue> stratificationValues,
+      JsonNode stratification,
+      int stratCount,
+      boolean measurePopulationBasis) {
+    String stratId = stratification.get("id").toString().replace("\"", "");
+    // We need to match up the stratifications to the groups and group
+    // population...
+    String stratName = "Strata-" + stratCount;
+    ArrayNode stratums = (ArrayNode) stratification.get("stratum");
+
+    List<JsonNode> stratumsList =
+        StreamSupport.stream(stratums.spliterator(), false)
+            .filter(
+                stratum -> {
+                  return stratum.findPath("value").findValue("text").asBoolean();
+                })
```

Comment:
Minor style simplication.
```suggestion
                stratum -> stratum.findPath("value").findValue("text").asBoolean())
```